node/src/main/scala/org/bitcoins/node/models/BroadcastAbleTransactionDAO.scala

```diff
@@ -26,4 +30,29 @@ final case class BroadcastAbleTransactionDAO(profile: JdbcProfile)(
     database.run(query.result).map(_.headOption)
   }
 
+  /** Table over TXs we can broadcast over the P2P network */
+  class BroadcastAbleTransactionTable(tag: Tag)
+      extends TableAutoInc[BroadcastAbleTransaction](tag, "broadcast_elements") {
+    private type Tuple = (DoubleSha256DigestBE, ByteVector, Option[Long])
+
+    private val fromTuple: (Tuple => BroadcastAbleTransaction) = {
+      case (txid, bytes, id) =>
+        val tx = Transaction.fromBytes(bytes)
+        require(tx.txId == txid.flip)
+        BroadcastAbleTransaction(tx, id)
+    }
+
+    private val toTuple: BroadcastAbleTransaction => Option[Tuple] = tx =>
+      Some(tx.transaction.txId.flip, tx.transaction.bytes, tx.id)
+
+    import org.bitcoins.db.DbCommonsColumnMappers._
+
+    def txid: Rep[DoubleSha256DigestBE] = column("txid", O.Unique)
+    def bytes: Rep[ByteVector] = column("tx_bytes")
+
+    def * : ProvenShape[BroadcastAbleTransaction] =
+      (txid, bytes, id.?) <>
+        (fromTuple, toTuple)
```

Comment:
In the `BlockHeaderDAO` you were able to get rid of the `fromTuple` and `toTuple` design, can you do that here and other places as well. Could then close #1137

Reply:
I'm not going to do that on this PR as the diff is already big enough, I don't think I got rid of it in `BlockHeaderDAO` it just didn't exist in the first place.